draft-wussler-openpgp-pqc.md

```diff
@@ -1028,27 +952,25 @@ scheme is as follows:
 
  3. Check that the own and the extracted algorithm ID match
 
- 4. Parse the `eccSecretKey` and `kyberSecretKey` from the algorithm specific
+ 4. Parse the `eccSecretKey` and `mlkemDecapsKey` from the algorithm specific
```

Comment:
Good catch. Changed `eccSecretKey` to `eccPrivateKey`

And yes, I adopted the terminology from ML-KEM specification. This is a remark I forgot to write in the PR description above. We should discuss that. I introduced this terminology in §5.1.2, saying that the encapsulation key is public and the decapsulation key is private keying material. I think since we have composite public and private keys which are composed of the ECC private/public and the ML-KEM Decapsulation/Encapsulation keys it does no harm to adopt the terminology of NIST.

Reply:
team agreed on this suggestion on 2023-09-11

Reply:
Team agrees on:
- eccPublicKey and eccSecretKey
- mlkemPublicKey and mlkemSecretKey
- ecdsaPublicKey and ecdsaSecretKey
- eddsaPublicKey and eddsaSecretKey

Todo @fluppe2